cfde_deriva/configs/portal/c2m2-level1-portal-model.json

```diff
@@ -2280,6 +2321,46 @@
             ],
             "missingValues": [ "" ],
             "primaryKey": "id"
+        },
+        "deriva": {
+           "source_definitions": {
+              "columns": true,
+              "fkeys": true,
+              "sources": {
+                 "S_subject": {
+                    "markdown_name": "Subject",
+                    "comment": "List of subjects referencing this taxonomy.",
+                    "source": [
+                        {"inbound": ["CFDE", "subject_species_species_fkey"]},
```

Comment:
while we use this derived subject-species table in our facets, we should probably skip this and go directly from the subject-role-taxonomy table
- inbound `subject_role_taxonomy_taxonomy_fkey`
- outbound `subject_role_taxonomy_subject_fkey`